dtfs-central-api/api-tests/wipeDB.js

```diff
@@ -1,7 +1,8 @@
 const db = require('../src/drivers/db-client');
 
 const wipe = async (collections) => {
-  const drop = async (collection) => new Promise(async (resolve, reject) => {
+  // eslint-disable-next-line no-async-promise-executor
+  const drop = async (collection) => new Promise(async (resolve) => {
```

Comment:
I think we can fix this lint error rather than disabling it if we change it to:
```
  const drop = async (collection) => new Promise((resolve) => {
    db.getCollection(collection).then((collection) => {
      collection.drop(() => { resolve() });
    });
  });
```

Reply:
Good shout, have refactored.